src/components/TextInput/TextInput.js

```diff
@@ -41,6 +42,7 @@ const InputElem = styled.input`
   border-radius: 0px;
   outline: none;
   font-size: 21px;
+  text-indent: ${props => (props.hideCursor ? '-9999em' : '')};
```

Comment:
`text-indent` can be removed. It's unused & was just an idea before creating the `DirectoryPicker` component.

I'll remove it.